core/math/a_star.h

```diff
@@ -50,8 +50,8 @@ class AStar3D : public RefCounted {
 		real_t weight_scale = 0;
 		bool enabled = false;
 
-		AHashMap<int64_t, Point *> neighbors = 4u;
```

Comment:
It is bizarre to me that this is even currently allowed, lol. Glad to see it removed